setup.py

```diff
@@ -24,6 +24,7 @@
     "pytest-black==0.3.12",
     "black==22.6.0",
     "django-environ==0.4.5",
+    "pyflakes==2.4.0",
```

Comment:
where is this used?

Reply:
pyflakes is used by pylama. I got errors about it when running pytest.

```
AttributeError: module 'pyflakes.messages' has no attribute 'RedefinedInListComp'
```

I found the issue at https://github.com/klen/pylama/issues/224. Follow the discussion to pin the pyflakes version and 
run pytest successfully.